py/visualize.py

```diff
@@ -51,33 +64,164 @@ def poll_vis(responses, poll_id, question_id=None, crosstab_variable="-"):
         & (responses.question_id == question_id)
         & (responses.xtab1_var == crosstab_variable)
     ]
+
+    # target_responses["question_text_wrap"] = ppd.plotly_wrap(target_responses.question_text.copy(),130)
+    # # question_text=target_responses["question_text_wrap"].iloc[0] 
+    # question_text=target_responses["question_text_wrap"].unique()[0] if target_responses.shape[0] > 0 else "No question text"
+
+    # ------------------ copied from plotly ------------------ #
+    #
+    response_set_df=target_responses[["response","response_order"]]
+
+    # response set in order
+    response_set = target_responses[["response","response_order"]].sort_values(by=["response_order"]).response.unique()
+    # create a list comprehsion to get the response labels in the same order as the response order
+    top_labels = [x.replace(" ","<br>") for x in target_responses.sort_values(by=["response_order"]).response.unique()]
+    # find the line length of the longest in top_labels
+    max_line_length = max([x.count("a") for x in top_labels])
+
+    # Define UBI Center colors
+    BLUE = "#1976D2"
+    DARK_BLUE = "#1565C0"
+    LIGHT_BLUE = "#90CAF9"
+    GRAY = "#BDBDBD"
+    BARELY_BLUE = "#E3F2FD"
+    # Define other colors
+    SUPER_DARK_GRAY = "#222222"
+    VERY_DARK_GRAY = "#333333"
+    DARK_GRAY = "#444444"
+    BLACK = "#000000"
+
+    colors=[
+        DARK_BLUE, 
+        BLUE, 
+        LIGHT_BLUE, 
+        BARELY_BLUE,
+        GRAY, 
+        ]
+
+    colors = colors[::-1]
+
+    #lighter grays, darker blues
+    colors = ["#494848", "#636363" ,"#B4B4B4", BLUE, DARK_BLUE]
+    # VERY_DARK_GRAY to VERY_LIGHT_GRAY
+    colors = [VERY_DARK_GRAY, DARK_GRAY, "#B4B4B4", BLUE, DARK_BLUE]
+    if len(top_labels) == 5:
+        colors = [VERY_DARK_GRAY, DARK_GRAY, GRAY, BLUE, DARK_BLUE]
+    elif len(top_labels) == 4:
+        colors = [VERY_DARK_GRAY, DARK_GRAY, BLUE, DARK_BLUE]
+    elif len(top_labels) == 3:
+        colors = [DARK_GRAY, GRAY, BLUE]
+    elif len(top_labels) == 2:
+        colors = [DARK_GRAY, BLUE]
     
-    target_responses["question_text_wrap"] = ppd.plotly_wrap(target_responses.question_text.copy(),130)
-    # question_text=target_responses["question_text_wrap"].iloc[0] 
-    question_text=target_responses["question_text_wrap"].unique()[0] if target_responses.shape[0] > 0 else "No question text"
-    
+    TEXTCOLORS = ['rgb(248, 248, 255)','rgb(248, 248, 255)','rgb(248, 248, 255)','rgb(248, 248, 255)','rgb(248, 248, 255)']
```

Comment:
```suggestion
    NEW_BLUE = 'rgb(248, 248, 255)'
    TEXT_COLORS = [NEW_BLUE] * 5
```
or something...please name the color descriptively (same goes for other `rgb` calls below) and include a comment as to why you need 5 of them